frontend/src/metabase-lib/lib/metadata/Metadata.ts

```diff
@@ -18,7 +19,8 @@ import Database from "./Database";
  */
 
 export default class Metadata extends Base {
-  databases: Database[];
```

Comment:
Pretty certain this type is incorrect -- the properties on `Metadata` like `databases` and `schemas` are all maps of ids to instances.